gremlin/values.yaml

```
@@ -114,6 +114,8 @@ gremlin:
     #   Gremlin's requirements (see gremlin.podSecurity.podSecurityPolicy and
     #   gremlin.podSecurity.securityContextConstraints)
     create: true
+    # Cloud IAM role to use to resolve any remote secrets
+    iamRole:
```

Perhaps something like the following to allow adding any arbitrary environment vars, such as
```yaml
env:
  add:
    - name: IAM_ROLE_ARN
      value: arn:aws-iso:iam::208767367117:role/gremlin-role
```

```suggestion
    iamRole:
  env:
    # gremlin.env.add - 
    # Specify any arbitrary enironment variables to pass to the Gremlin Agent daemonset.
    add:
```